nltk/classify/rte_classify.py

```diff
@@ -70,8 +70,8 @@ def __init__(self, rtepair, stop=True, lemmatize=False):
         self.hyp_words = set(self.hyp_tokens)
 
         if lemmatize:
-            self.text_words = set(lemmatize(token) for token in self.text_tokens)
```

Comment:
hi @jnothman ! I agree with the second part of your statement but I'm not sure about the first one: this is essentially `set(list generating expression)` which I think is correct and was the original intention?
I'm not very familiar with that code and I might be totally off though!